packages/design-system-tailwind-preset/src/colors.ts

```diff
@@ -0,0 +1,56 @@
+export const colors = {
```

Comment:
Break out colors into separate PR

Reply:
Keeping colors to insure initial testing and going to work on tests in a different PR https://github.com/MetaMask/metamask-design-system/issues/48